packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -41,8 +41,14 @@
 			</tera-drilldown-section>
 		</div>
 		<div :tabName="StratifyTabs.Notebook">
-			<tera-drilldown-section>
-				<p class="mt-3 ml-4">Code Editor - Python</p>
+			<tera-drilldown-section class="notebook-section">
+				<tera-notebook-jupyter-input
```

Comment:
I am confused why you do not need `Suspense`

Either its not required or I am unable to click fast enough to send something before the kernelManager is built so it doesnt appear to be a problem.